crates/dprint/Cargo.toml

```diff
@@ -28,6 +28,7 @@ twox-hash = "1.6.0"
 url = "2.1.1"
 wasmer = "=1.0.0"
 zip = "0.5.8"
+wild = "2.0.4"
```

Comment:
An issue might be that it uses a different globbing dependency (https://crates.io/crates/glob) than what the rest of dprint uses (https://crates.io/crates/globwalk) and there seems to be some behaviour differences between the two. Perhaps this is not an issue though since it will cause the same behaviour on windows and unix for unquoted globbing.

Does this ignore quoted globs? From what I can tell from the source it seems to. If so, this looks good!

Reply:
'wild' does ignore quoted globs.

Since 'wild' only affects the Windows command line args, I think the slight discontinuity between 'glob' and 'globwalk' is reasonable for the benefit that it brings. I don't think it will cause many user issues. That being said, I am hoping that Rust starts coming together with a more robust, common/std-supported glob/globwalk in the near future. There is some discussion about it (https://github.com/rust-lang-nursery/glob/issues/59), but it's slow going at the moment.

Reply:
@rivy cool. Yeah I did some testing after merging and it looks good! It's nice to have them acting the same from the command line now.